src/main/java/io/cdap/delta/bigquery/BigQueryAssessor.java

```diff
@@ -61,6 +65,20 @@ public TableAssessment assess(StandardizedTableDetail tableDetail) {
     List<Problem> problems = new ArrayList<>();
     String dbName = tableDetail.getDatabase();
     String tableName = tableDetail.getTable();
+
+
+    if (tables.containsKey(tableName)) {
+      // TODO support same table name in different schema
+      if (!Objects.equals(tables.get(tableName), tableDetail.getSchemaName())) {
+        problems.add(new Problem("Duplicate Table Name", String.format("Duplicated table name %s detected. BigQuery " +
```

Comment:
nit: Message can be improved a little bit to include schemaNames. 
How about 
```
Table with name '%s' found in two different schemas, '%s' and '%s'. BigQuery target requires table names to be unique across the schemas. Please select table from one of the schemas only for replication.
```

Reply:
done :)